### ISSUE TITLE
Remapping 'L' & 'R' buttons won't work for Switch Pro Controller

### ISSUE THREAD

**Issue by Matthbo** [libretro/dolphin#96]
I use a PowerA GameCube controller made for the Nintendo Switch, this controller works / is seen as a Switch Pro Controller.

In the user binds I have set 'L' to 'ZL'(seen as btn 6), 'R' to 'ZR'(seen as btn 7), 'L2' to 'L'(seen as btn 4) & 'R2' to 'R'(seen as btn 5).
While testing a Zelda Wind Waker Randomizer I noticed that the 'L(seen as btn 4)' & 'R(seen as btn 5)' buttons still work as L & R buttons ingame, so I tried to remap the buttons for this content but it seems like they are permanently mapped.

In conclusion, the content loaded won't listen to user bindings or controller settings for this content for the 'L' & 'R' buttons on Switch Pro Controller
